Pegasus/sources/particleforcegenerator.cpp

```diff
@@ -259,7 +256,7 @@ void pegasus::BlobForceGenerator::updateForce(const pegasus::Particle::Ptr& part
 
     // If the particle is the head, and we've got a join count, then float it.
     if (particle == particles.front() && joinCount > 0 && maxFloat > 0) {
-        auto force = real(joinCount / maxFloat) * floatHead;
+        auto force = double(joinCount / maxFloat) * floatHead;
```

Comment:
Without casting both `joinCount` and `maxFloat` you will get only the result of integer division (without decimal part). But maybe it was meant to be like that.